include/torch-mlir/Conversion/Utils/Utils.h

```diff
@@ -100,6 +100,17 @@ void computeBroadcastShape(ConversionPatternRewriter &rewriter, Location loc,
                            SmallVector<int64_t> &resultShape,
                            SmallVector<Value> &resultShapeValue);
 
+// Helper function for expand index tentors, the following expandSizes().
```

Comment:
It don't understand this sentence. Could you please expand the explanation?

Reply:
Done